### PR TITLE
Support PS comments and skip invalid rows rather than aborting

### PR DESCRIPTION
**Pull request by matteosb**

### CODE REVIEW

File: enip_backend/comments_gsheet_sync/run.py [hunk 1]
```diff
@@ -38,28 +39,38 @@ def _get(row, header, validate_truthy=True):
...
+            yield {
+                "ts": TS_TZ.localize(ts),
+                "submitted_by": _get(r, NAME_HEADER),
+                "office_id": office_id,
+                "race": race,
+                "title": _get(r, TITLE_HEADER),
+                "body": _get(r, BODY_HEADER),
+            }
+    except Exception as err:
+        logging.error("Skipping. Exception: {}".format(err))
```
Comment (benweissmann) [inline enip_backend/comments_gsheet_sync/run.py:75]: Should also call `sentry_sdk.capture_exception(err)`

**Review by benweissmann — APPROVED** [review]
looks good, just one flag

File: enip_backend/comments_gsheet_sync/run.py (hunk 1 above)
Comment (matteosb) [reply enip_backend/comments_gsheet_sync/run.py:75]: good idea, updated

**Review by jasonkb — COMMENTED** [review]
brilliant tyty!

**Pull request merged**